### PR TITLE
[WIP] feat(doc): #361 document node

### PR DESCRIPTION
**Pull request by dsalaza4**
- Document node args

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2186,6 +2186,165 @@ $ m . /example
...
+#### makeNodeInterpreter
+
+Get a specific [Node.js][NODE_JS] version.
+Useful for passing a node version
+to either `makeNodeEnvironment`
+or `makeNodeModules`.
+
+Types:
+
+- makeNodeInterpreter (`function version -> package`):
```
Comment (kamadorueda) [inline README.md:2201]:
```suggestion
- makeNodeInterpreter (`function str -> package`):
```

File: README.md [hunk 2]
```diff
@@ -2186,6 +2186,165 @@ $ m . /example
...
+
+    - version (`enum [ "10" "12" "14" "16" ]`):
```
Comment (kamadorueda) [inline README.md:2203]:
```suggestion
    - (`enum [ "10" "12" "14" "16" ]`):
```

File: README.md [hunk 3]
```diff
@@ -2186,6 +2186,165 @@ $ m . /example
...
+    ls __argHello__
+  '';
+  name = "example";
+}
+```
+
+```bash
+$ m . /example
+
+    node_modules  package.json  package-lock.json  requirements
```
Comment (kamadorueda) [inline README.md:2286]: this should be returning the node_modules/* instead of the parent dir of node_modules

File: README.md [hunk 4]
```diff
@@ -2186,6 +2186,165 @@ $ m . /example
...
+
+#### makeNodeEnvironment
+
+Source a `makeNodeModules` environment
+using `makeSearchPaths`.
+It appends:
+
+- `node` to `PATH`.
+- `node_modules/.bin` to `PATH`.
+- `node_modules` to `NODE_PATH`.
```
Comment (kamadorueda) [inline README.md:2297]:
```suggestion
- `node_modules` to [NODE_PATH][NODE_PATH].
```

**Review by kamadorueda — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (dsalaza4) [reply README.md:2323]: Will implement this in another MR.

**Pull request merged**